examples/practice-typescript/src/scripts/app.tsx

```diff
@@ -50,6 +51,22 @@ const App = () => {
         <Button text="Edit" styleClass="btn-action" />
         <Button text="Delete" styleClass="btn-action" />
       </div>
+      <table>
+        <TableRow
+          id="1"
+          email="kim@gmail.com"
+          firstName="tran "
+          lastName="kim"
+          phoneNumber="0132456789"
+        />
+         <TableRow
+          id="2"
+          email="kim@gmail.com"
+          firstName="tran "
+          lastName="kim"
+          phoneNumber="0132456789"
+        />
```

Comment:
using map rendering

Reply:
Fixed and I using map rendering in next PR TableBody